AWS-CodePipeline/sam-monorepo-pipeline-template/{{cookiecutter.outputDir}}/pipeline/assume-role.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Personally, I believe this is a worse UX. The current method neatly bundles the Pipeline infrastructure (scripts, stack, etc) with the application subfolder (created by `sam init`). There is also the case that one might _not_ want to share the bootstrapped structure.

That being said, I suppose it is possible to change the template enough so that files are generated all inside the folder specified by `monorepo_project_subfolder`, but that would be a significant change.

@aahung Would you be OK with progressing with this as is, and I can work on a separate PR with this change?